greenwood.config.js

```diff
@@ -20,6 +20,61 @@ module.exports = {
     { rel: 'icon', href: FAVICON_HREF },
     { name: 'google-site-verification', content: '4rYd8k5aFD0jDnN0CCFgUXNe4eakLP4NnA18mNnK5P0' }
   ],
+  optimization: {
+    splitChunks: {
+      chunks: 'all',
+      minSize: 30000,
+      maxSize: 0,
+      minChunks: 1,
+      maxAsyncRequests: 6,
+      maxInitialRequests: 4,
+      automaticNameDelimiter: '~',
+      automaticNameMaxLength: 30,
+      cacheGroups: {
```

Comment:
yeah, We should be doing this in _webpack.config.common.js_ for the user by default.

And also do some sort of dynamic creation of `cacheGroups` here like we do in _webpack.config.common.js_ with `NormalModuleReplaclementPlugin`.

Reply:
I was thinking along the lines of similar to a plugin, if a developer wants custom split routes they can configure it however they please.